src/rsz/src/RepairSetup.hh

```diff
@@ -168,7 +178,10 @@ private:
   const MinMax *min_;
   const MinMax *max_;
 
-  sta::UnorderedMap<LibertyCell *, sta::LibertyPortSet> equiv_pin_map_;
+  sta::UnorderedMap<std::pair<LibertyCell*, LibertyPort*>,
```

Comment:
Why do you need the pair?  A LibertyPort can only belong to one LibertyCell so it seems redundant to mention the cell and would remove the need for Boost.

Reply:
That does seem correct, but it does seem a bit less readable to my eye. The reader would need to know that I guess. I can make the change, but I'll probably add a comment so it's clear.

Reply:
This is less readable to my eye because I wonder why the cell is there - could it not be the cell associated with this port? I guess it depends on experience.   I'd prefer the change with a comment